charts/stable/home-assistant/values.yaml

```diff
@@ -138,8 +138,11 @@ configmap:
         http:
           use_x_forwarded_for: true
           trusted_proxies:
+            {{- if hasKey .Values "ixChartContext" }}
+            - {{ .Values.ixChartContext.kubernetes_config.cluster_cidr }}
+            {{- else }}
             - 172.16.0.0/16
-
+            {{- end }}
```

Comment:
Preferably we allow "trusted proxies" to be exposed in values.yaml, but not in questions.yaml (as it's not usable on SCALE).

Then inside the "else" we range through the "trusted proxies" list in values.yaml, instead of using an arbitrary value.

---

However, in general: Now that we officially(!) do not support native helm anymore, we should default more of these (nextcloud for example afaik) to `{{ .Values.ixChartContext.kubernetes_config.cluster_cidr }}`.

So we might rather want to do all of those at once (just search for 172.16 really.)

Reply:
Replaced the hardcoded else on HA.

But a quick look showed that on other apps we expose the option in the GUI to change.
Making it a bit trickier than just find and replace.

Lets create an issue for tracking, and merge this one as this is currently "no worky" for people with different cluster cidr